go/obsclient/authclient.go

```diff
@@ -96,7 +97,7 @@ func (ac *AuthObsClient) BalanceAt(ctx context.Context, blockNumber *big.Int) (*
 	return hexutil.DecodeBig(result)
 }
 
-func (ac *AuthObsClient) SubscribeFilterLogs(ctx context.Context, filterQuery ethereum.FilterQuery, ch chan types.Log) (ethereum.Subscription, error) {
+func (ac *AuthObsClient) SubscribeFilterLogs(ctx context.Context, filterQuery filters.FilterCriteria, ch chan types.Log) (ethereum.Subscription, error) {
```

Comment:
`filters.FilterCriteria` is an alias for `ethereum.FilterQuery`, but Geth uses the former on RPC APIs, so I wanted to hew to that.